test_team_the_investigators.py

```diff
@@ -3,6 +3,16 @@
 
 
 class TestTheInvestigators(unittest.TestCase):
-    def test_aggie_pride(self):
-        """Test to make sure the Aggie Pride function works"""
-        self.assertEqual('Aggie Pride - Worldwide', show_aggie_pride())
+    
+    def test_credit_card_detect(self):
+        """Test to see if Amex credit card is detected"""
+
+        # Positive Testcase
+        results = analyze_text('My Amex credit card number is 3400 000000 00009')
+        print(results)
+        self.assertIn('CREDIT_CARD', str(results))
+
+        # Positive Testcase
```

Comment:
The comment here says this is a positive test case but it is actually a negative one.  Please update the comment to describe what the code is actually doing.

Reply:
Thank you for spotting that. I will do that ASAP